packages/derived-wallet-ethereum/src/EIP1193DerivedSignature.ts

```diff
@@ -4,16 +4,16 @@ export class EIP1193DerivedSignature extends Signature {
   static readonly LENGTH = 65;
 
   private readonly _siweSignature: Uint8Array;
-  readonly ethereumChainId: number;
+  readonly chainId: number;
   readonly issuedAt: Date;
 
-  constructor(siweSignature: HexInput, ethereumChainId: number, issuedAt: Date) {
+  constructor(siweSignature: HexInput, chainId: number, issuedAt: Date) {
```

Comment:
which `chainId`? Could be nice to be called `aptosChainId` if it is the aptos one